src/admin/views/Dashboard.tsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
 
-const Dashboard = () => <h1 className="c-h2">Dashboard</h1>;
+import { AdminLayout, AdminLayoutBody } from '../layouts';
+
+const Dashboard = () => (
+	<AdminLayout pageTitle="Dashboard">
```

Comment:
Moet er een dashboard zijn of gaan we by default naar het eerste item in de sidebar?
Ik stel de vraag aan leen: https://district01.atlassian.net/browse/AVO2-206

Reply:
Antwoord van leen:
> Voorzie maar een dashboard, voorlopig gewoon welkom boodschap zetten. Kan later nog betekenis krijgen. Maar als je hier extra werk aan hebt kan dit perfect wachten tot later